shell/common/asar/archive_win.cc

```diff
@@ -0,0 +1,179 @@
+// Copyright 2023 Slack Technologies, Inc.
+// Contributors: Weiyun Dai (https://github.com/WeiyunD/), Andrew Lay
+// (https://github.com/guohaolay) Use of this source code is governed by the MIT
+// license that can be found in the LICENSE file.
+
+#include "shell/common/asar/archive.h"
+
+#include <algorithm>
+#include <sstream>
+
+#include "base/json/json_reader.h"
+#include "base/logging.h"
+#include "base/no_destructor.h"
+#include "base/strings/string_util.h"
+#include "base/strings/string_util_win.h"
+#include "base/strings/utf_string_conversions.h"
+#include "shell/common/asar/asar_util.h"
+
+using namespace std;
+using namespace base;
```

Comment:
Avoid `using namespace` in Electron code.

Reply:
Updated.

I understand why we should avoid using namespace in header files, but curious to know why also avoid doing so in source files.

Reply:
The main reason is that it makes it harder to know where a symbol comes from when reading the file. It's also against the Chromium/Google C++ style guide to do so: https://google.github.io/styleguide/cppguide.html#Namespaces